cypress/support/step_definitions/genericSteps/genericSteps.ts

```diff
@@ -74,7 +74,7 @@ Then('I see the {string} section heading', (sectionName: string) => {
   cy.contains('h2', sectionName);
 });
 Then('I see the {string} link under the {string} section', (linkName: string, sectionName: string) => {
-  cy.contains('h2', sectionName).next().contains('a', linkName);
+  cy.contains('h2', sectionName).next().contains('li', linkName);
```

Comment:
This should be in a different step please, one for the link (a) and one for just text (li)

Reply:
This is not misleading other tests. All tests passed

Reply:
more so for the readability of the step, doing what it says its doing :)